src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -111,13 +124,91 @@ public static void Https_SecModeTrans_ClientCredTypeNone_ServerCertValModePeerOr
         string testString = "Hello";
         ChannelFactory<IWcfService> factory = null;
         IWcfService serviceProxy = null;
+        CommunicationException communicationException = null;
 
         try
         {
             // *** SETUP *** \\
             BasicHttpsBinding binding = new BasicHttpsBinding(BasicHttpsSecurityMode.Transport);
             binding.Security.Transport.ClientCredentialType = HttpClientCredentialType.None;
-            endpointAddress = new EndpointAddress(new Uri(Endpoints.Https_SecModeTrans_ClientCredTypeNone_ServerCertValModePeerTrust_Address));
+            endpointAddress = new EndpointAddress(new Uri(
+                                Endpoints.Https_SecModeTrans_ClientCredTypeNone_ServerCertValModePeerOrChainTrust_Address));
```

Comment:
The name of the endpoint caused confusion, so I am renaming.  That endpoint uses a cert guaranteed to be chain trusted and also not in TrustedPeople.  I'll remove the "PeerOr" from the name of the endpoint and its matching service host.

---
In reply to: [71626389](https://github.com/dotnet/wcf/pull/1382#discussion_r71626389) [](ancestors = 71626389)